lib/fog/compute/google/models/network.rb

```diff
@@ -36,12 +36,10 @@ def destroy(async = true)
           operation
         end
 
-        # Returns a ready API structure for insert_instance, used in insert_server request.
-        def get_as_interface_config(access_config = nil)
-          network_interface = { "network" => self_link }
-          network_interface["accessConfigs"] = [access_config] if access_config
-          network_interface
```

Comment:
I am not sure if it is a good way to add subnetwork setting in the method get_as_interface_config which is the method of class network and not subnetwork. That's why I have moved network_interface outside. I will think about how to make it better.